### PR TITLE
Cp 147 refactor scheduled tasks

### PR DESCRIPTION
**Pull request by zuming-acn**

### CODE REVIEW

File: cockpit_api/pom.xml [hunk 1]
```diff
@@ -41,6 +41,10 @@
 					<groupId>org.junit.vintage</groupId>
 					<artifactId>junit-vintage-engine</artifactId>
 				</exclusion>
+				<exclusion>
+					<groupId>org.skyscreamer</groupId>
```
Comment (RihabRJAB) [inline cockpit_api/pom.xml:45]: do we need this dependency ?

File: cockpit_api/src/main/java/com/cockpit/api/model/dto/JiraSprintDTO.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    private String state;
+    private Date startDate;
+    private Date completeDate;
+
+    public Integer getOriginBoardId() {
+        return originBoardId;
+    }
+
+    public Date getEndDate() {
+        return endDate;
```
Comment (RihabRJAB) [inline cockpit_api/src/main/java/com/cockpit/api/model/dto/JiraSprintDTO.java:22]: JiraSprintDto do we still use this class ?

Comment (zuming-acn) [reply cockpit_api/src/main/java/com/cockpit/api/model/dto/JiraSprintDTO.java:22]: no, removed

File: cockpit_api/pom.xml (hunk 1 above)
Comment (zuming-acn) [reply cockpit_api/pom.xml:45]: no more need; Kojak added it for his unit tests, removed

**Pull request merged**